drivers/src/pcr_log.rs

```diff
@@ -29,15 +29,16 @@ pub enum PcrLogEntryId {
     AntiRollbackDisabled = 3,  // data size = 1 byte
     VendorPubKeyHash = 4,      // data size = 48 bytes
     OwnerPubKeyHash = 5,       // data size = 48 bytes
-    EccVendorPubKeyIndex = 6,  // data size = 1 byte
-    FmcTci = 7,                // data size = 48 bytes
-    FmcSvn = 8,                // data size = 1 byte
-    FmcFuseSvn = 9,            // data size = 1 byte
-    LmsVendorPubKeyIndex = 10, // data size = 1 byte
-    RomVerifyConfig = 11,      // data size = 1 byte
-    StashMeasurement = 12,     // data size = 48 bytes
-    RtTci = 13,                // data size = 48 bytes
-    FwImageManifest = 14,      // data size = 48 bytes
+    OwnerPubKeysInFuses = 6,   // data size = 1 bytes
```

Comment:
Can we rename this to 'OwnerPubKeysHashInFuses'?

Reply:
Done